Project-1/README.md

```diff
@@ -67,9 +183,99 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
   (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+ (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+ (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (4)¬(Pa ∧ Qa)
+ (5)¬(¬Pa ∨ ¬Qa)
+ (6).¬¬Pa
+ (7)¬¬Qa
+ (8)Qa
+ (9) Pa
+ (10) ¬Pa	(11) ¬Qa
+        x	       x
+ 
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+  (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (4) ¬(Pa ∧ Qa)
+  (5) ¬(¬Pa ∨ ¬Qa)
+  (6) ¬¬Pa
+  (7) ¬¬Qa
+  (8) Qa
+  (9) Pa
+  (10) ¬Pa	(11) ¬Qa
+        x	      x
+  
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  
+  (1) ¬∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  (2) ¬∀y((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (3) ¬((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (4) ¬Pa ∨ ¬Qa
+  (5) ¬¬(Pa ∧ Qa)
+  (6) Pa ∧ Qa
+  (7) Pa
+  (8) Qa
+  (9) ¬Pa	(10) ¬Qa
+     x		     x
+  
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+ (1) ¬∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+ (2) ¬∀y((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (3) ¬((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (4) ¬Pa ∧ ¬Qa
+ (5) ¬¬(Pa ∨ Qa)
+ (6) Pa ∨ Qa
+ (7) ¬Pa
+ (8) ¬Qa
+ (9) Pa		(10) Qa
+     x		     x
+     
 ```
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
 
+¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+(1) ¬(Px ∧ Qx)		assumption
+(2) ¬(¬Px ∨ ¬Qx)	assumption
+(3) (Px ∧ Qx)		2, negated disjunction
+(4) ¬(Px ∧ Qx)		1,3 contradiction
+
+∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+(1) ¬(Px ∨ Qx)		assumption
+(2) (¬Px ∧ ¬Qx)		assumption
+(3) (Px ∨ Qx)		2, negated conjunction
+(4) ¬(Px ∨ Qx)		1,3 contradiction
+
+∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+(1) (¬Px ∨ ¬Qx)		assumption
+(2) ¬¬(Px ∧ Qx)		assumption
+(3) (Px ∧ Qx)		2, double negation
+(4) Px			3, conjunction
+(5) Qx			3, conjunction
+(6) (¬Px ∨ ¬Qx)		4,5,6 contradiction
+
+∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+(1) (¬Px ∧ ¬Qx)		assumption
+(2) ¬¬(Px ∨ Qx)		assumption
+(3) (Px ∨ Qx)		2, double negation
+(4) ¬Px			1, conjunction
+(5) ¬Qx			1, conjunction
+(6) Px			3, disjunction
+(7) ¬Qx			3, disjunction
+(8) ¬Px			3, disjunction
+(9) Qx			3, disjunction
+(10) (¬Px ∧ ¬Qx)	6,7,8,9 contradiction
+
 10. Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs. 
+
+Both proofs deduce some kind of contradiction, but in different ways.  8a is a longer proof than 9a. Feedback from Josh (thank you!) also states that 9a does not require us to go down to the atomic parts of the formula, and stops at more complex contradictions.
```

Comment:
Why do you think one is called "natural" deduction and the other is called 'tableau'?

Reply:
Natural thinking, next step entailed by the previous step, reasoning in order of already proved truths and falsities

Tableau, "a graphic description or representation," (according to Merriam Webster), the x's at the bottom indicate the branches which reveal a contradiction (in tree-graph form)